end2end-tests/models-jackson/src/test/resources/enums/enum_ref.json

```diff
@@ -0,0 +1,8 @@
+{
+  "array_of_enums" : null,
+  "inlined_enum" : null,
+  "inlined_extensible_enum" : null,
+  "enum_ref" : "one",
+  "extensible_enum_ref" : null,
+  "list_enums" : null
```

Comment:
do you need all the nulls in these examples? Might be cleaner to just leave them absent